ads_plugin.py

```diff
@@ -642,15 +643,8 @@ def on_compute_morphometrics_button(self, event):
             except IOError:
                 wx.LogError("Cannot save current data in file '%s'." % pathname)
 
-        # Create the axon coordinate array
-        mean_diameter_in_pixel = np.average(x['axon_diam']) / pixel_size
-        axon_indexes = np.arange(x.size)
-        number_array = postprocessing.generate_axon_numbers_image(axon_indexes, x['x0'], x['y0'],
-                                                                  tuple(reversed(axon_array.shape)),
-                                                                  mean_diameter_in_pixel)
 
-        # Load the axon coordinate image into FSLeyes
-        number_outfile = self.ads_temp_dir / "numbers.png"
+        number_outfile = Path(pathname).parents[0] / "numbers.png"
```

Comment:
I wasn't able to test this PR for FSLeyes because of issue #521.  

Looking by the code, I'm not sure whether the FSLeyes also generates two files: `_numbers_seg.png` and `_colored_seg.png`? The behaviour should be consistent with FSLeyes and CLI.